lib/bitstyles_phoenix/component/flash.ex

```diff
@@ -0,0 +1,129 @@
+  """
+
+  story("Flash brand 1", '''
+      iex> assigns = %{}
+      iex> render(~H"""
+      ...>   <.ui_flash variant="brand-1">
+      ...>     Something you may be interested to hear
+      ...>   </.ui_flash>
+      ...> """)
+      """
+      <div aria-live="polite" class="u-padding-l-y a-flash a-flash--brand-1">
+        <div class="a-content u-flex u-items-center u-font--medium">
+          Something you may be interested to hear
+        </div>
+      </div>
+      """
+  ''')
+
+  story("Flash success", '''
+      iex> assigns = %{}
+      iex> render(~H"""
+      ...>   <.ui_flash variant="positive">
+      ...>     Saved successfully
+      ...>   </.ui_flash>
+      ...> """)
+      """
+      <div aria-live="polite" class="u-padding-l-y a-flash a-flash--positive">
+        <div class="a-content u-flex u-items-center u-font--medium">
+          Saved successfully
+        </div>
+      </div>
+      """
+  ''')
+
+  story("Flash warning", '''
+      iex> assigns = %{}
+      iex> render(~H"""
+      ...>   <.ui_flash variant="warning">
+      ...>     Saved with errors
+      ...>   </.ui_flash>
+      ...> """)
+      """
+      <div aria-live="polite" class="u-padding-l-y a-flash a-flash--warning">
+        <div class="a-content u-flex u-items-center u-font--medium">
+          Saved with errors
+        </div>
+      </div>
+      """
+  ''')
+
+  story("Flash danger", '''
+      iex> assigns = %{}
+      iex> render(~H"""
+      ...>   <.ui_flash variant="danger">
+      ...>     Saving failed
+      ...>   </.ui_flash>
+      ...> """)
+      """
+      <div aria-live="polite" class="u-padding-l-y a-flash a-flash--danger">
+        <div class="a-content u-flex u-items-center u-font--medium">
+          Saving failed
+        </div>
+      </div>
+      """
+  ''')
+
+  story("Custom attributes and classes", '''
+      iex> assigns = %{}
+      iex> render(~H"""
+      ...>   <.ui_flash variant="brand-1" data-foo="bar" class="extra-class" content_class="extra-inner-class">
+      ...>     Saving failed
+      ...>   </.ui_flash>
+      ...> """)
+      """
+      <div aria-live="polite" class="u-padding-l-y a-flash a-flash--brand-1 extra-class" data-foo="bar">
+        <div class="a-content u-flex u-items-center u-font--medium extra-inner-class">
+          Saving failed
+        </div>
+      </div>
+      """
+  ''')
+
+  def ui_flash(assigns) do
+    variant = assigns[:variant]
+
+    class =
+      classnames([
+        "u-padding-l-y a-flash",
+        {"a-flash--#{variant}", variant != nil},
+        assigns[:class]
+      ])
+
+    content_class =
+      classnames(["a-content u-flex u-items-center u-font--medium", assigns[:content_class]])
+
+    extra = assigns_to_attributes(assigns, [:class, :content_class, :variant])
+    assigns = assign(assigns, class: class, content_class: content_class, extra: extra)
+
+    ~H"""
+    <div aria-live="polite" class={@class} {@extra}>
+      <div class={@content_class}>
```

Comment:
This deviates a bit from what is in the storybook currently


```
      <div aria-live="polite">
        <div class="u-padding-l-y a-flash--brand-1">
          <div class="a-content u-flex u-items-center u-font--medium">
            <svg width="14" height="14" class="a-icon a-icon--l u-flex-shrink-0 u-margin-m-right" aria-hidden="true" focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
              <use xlink:href="${icons}#icon-info-circle"></use>
            </svg>
            Something you may be interested to hear
          </div>
        </div>
      </div>
```

https://bitcrowd.github.io/bitstyles/?path=/docs/ui-flashes--flash-brand-1

just noticed, not sure what the differences mean

Reply:
You mean the nesting? I don;t think it makes a difference though...